app/controllers/v0/profile/valid_va_file_numbers_controller.rb

```diff
@@ -6,13 +6,15 @@ class ValidVaFileNumbersController < ApplicationController
       before_action { authorize :bgs, :access? }
 
       def show
-        service = BGS::PeopleService.new(current_user)
-        response = service.find_person_by_participant_id
-
-        render(
-          json: valid_va_file_number_data(response),
-          serializer: ValidVaFileNumberSerializer
-        )
+        render json: {
+    "data": {
+        "id": "",
+        "type": "valid_va_file_number",
+        "attributes": {
+            "valid_va_file_number": true
+        }
+    }
+}.to_json
       end
```

Comment:
Looks like some formatting issues here w/ indentation.

Reply:
This was not supposed to be in the PR, I removed it